### PR TITLE
Project Documentation(Ioana_maria.cristea)

### PR DESCRIPTION
**Pull request by Ella23411**
### Pull Request Overview

This pull request adds the project documentation (Smart Fan)

### Author

Signed-off-by: Cristea Ioana-Maria <cristeaioanamaria1@gmail.com>

### CODE REVIEW

File: docs/project/2024/Ioana_maria.cristea/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+| [DC Motor]() | DC Motor| [3,50 RON](https://www.emag.ro/motor-electric-dc-3-6v-ajmaker-8000rpm-6v-450-ma-25-x-20-x15-mm-gri-z000687/pd/DSFQLHYBM/?cmpid=93116&utm_source=google&utm_medium=cpc&utm_campaign=(RO:eMAG!)_3P_NO_SALES_%3e_Jucarii_hobby&utm_content=111476631565&gad_source=1&gclid=CjwKCAjw88yxBhBWEiwA7cm6pRCxklxkpkhCiPunMVr4_dPU2-iwMSfpSF-6S8OcYMyXWCNHiOyHPxoCQGIQAvD_BwE) |
+
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/Ioana_maria.cristea/index.md:95]: Do you need these libraries?

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/Ioana_maria.cristea/index.md (hunk 1 above)
Comment (Ella23411) [reply docs/project/2024/Ioana_maria.cristea/index.md:95]: Hi, no sorry I thought that we had to change those for when we have the software milestone
I'll delete them for now.

File: docs/project/2024/Ioana_maria.cristea/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+| [BMP280](https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmp280-ds001.pdf) | Temperature Sensor| [8,49 RON](https://www.optimusdigital.ro/ro/senzori-senzori-de-presiune/1666-modul-senzor-de-presiune-barometric-bmp280.html?search_query=bmp280&results=11) |
+| [Active Buzzer]() | Active Buzzer | [1,49 RON](https://www.optimusdigital.ro/ro/audio-buzzere/635-buzzer-activ-de-3-v.html?search_query=buzzer+activ&results=18) |
+| [LCD Display](http://www.lcdwiki.com/res/MSP1804/1.8inch_SPI_Module_MSP1804_User_Manual_EN.pdf) | LCD Display| [30 RON](https://www.optimusdigital.ro/ro/optoelectronice-lcd-uri/12392-modul-lcd-de-18-cu-spi-i-controller-st7735-128x160-px.html?search_query=lcd+st773&results=8) |
+| [DC Motor]() | DC Motor| [3,50 RON](https://www.emag.ro/motor-electric-dc-3-6v-ajmaker-8000rpm-6v-450-ma-25-x-20-x15-mm-gri-z000687/pd/DSFQLHYBM/?cmpid=93116&utm_source=google&utm_medium=cpc&utm_campaign=(RO:eMAG!)_3P_NO_SALES_%3e_Jucarii_hobby&utm_content=111476631565&gad_source=1&gclid=CjwKCAjw88yxBhBWEiwA7cm6pRCxklxkpkhCiPunMVr4_dPU2-iwMSfpSF-6S8OcYMyXWCNHiOyHPxoCQGIQAvD_BwE) |
+
```
Comment (irina-b-dev) [inline docs/project/2024/Ioana_maria.cristea/index.md:87]: Please add servo motor in the table.

File: docs/project/2024/Ioana_maria.cristea/index.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://www.youtube.com/watch?v=8kfROSQYDWI)
```
Comment (irina-b-dev) [inline docs/project/2024/Ioana_maria.cristea/index.md:101]:
```suggestion
1. [Youtube - Raspberry Pi Pico Temperature Controlled Fan Speed](https://www.youtube.com/watch?v=8kfROSQYDWI)
```

File: docs/project/2024/Ioana_maria.cristea/index.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+2. [link](https://projects.raspberrypi.org/en/projects/build-your-own-weather-station/2)
```
Comment (irina-b-dev) [inline docs/project/2024/Ioana_maria.cristea/index.md:102]: This link could be helpful but it shows how to operate with the BMP280 sensor on Raspberry Pi microprocessor, not the Raspberry Pi Pico microcontroller. If you choose to make your project in Embassy it is best to use their documentation and libraries.

**Comment by alexandruradovici** [comment]
The build fails as you have a link that is broken.

File: docs/project/2024/Ioana_maria.cristea/index.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-Ella23411
+
+:::
+
+## Description
+
+A fan that monitors and displays the temperature and pressure in the room on a LCD screen, then activates the fan after a certain threshold :
+
+
+  + The system utilizes a sensor to detect both temperature and pressure.
```
Comment (alexandruradovici) [inline docs/project/2024/Ioana_maria.cristea/index.md:16]: Is this formatted correctly?

Comment (Ella23411) [reply docs/project/2024/Ioana_maria.cristea/index.md]: It is ok now ?

Comment (Ella23411) [reply docs/project/2024/Ioana_maria.cristea/index.md]: Is it ok now?

Comment (alexandruradovici) [reply docs/project/2024/Ioana_maria.cristea/index.md]: If it renders corectly yes.

**Review by alexandruradovici — DISMISSED** [review]

File: docs/project/2024/Ioana_maria.cristea/index.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+| [BMP280](https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmp280-ds001.pdf) | Temperature Sensor| [8,49 RON](https://www.optimusdigital.ro/ro/senzori-senzori-de-presiune/1666-modul-senzor-de-presiune-barometric-bmp280.html?search_query=bmp280&results=11) |
+| [Active Buzzer] | Active Buzzer | [1,49 RON](https://www.optimusdigital.ro/ro/audio-buzzere/635-buzzer-activ-de-3-v.html?search_query=buzzer+activ&results=18) |
+| [LCD Display](https://www.waveshare.com/datasheet/LCD_en_PDF/LCD1602.pdf) | LCD Display| [30 RON](https://www.optimusdigital.ro/ro/optoelectronice-lcd-uri/62-lcd-1602-cu-interfata-i2c-si-backlight-galben-verde.html?search_query=lcd+1602&results=17) |
+| [DC Motor] | DC Motor| [3,50 RON](https://www.emag.ro/motor-electric-dc-3-6v-ajmaker-8000rpm-6v-450-ma-25-x-20-x15-mm-gri-z000687/pd/DSFQLHYBM/?cmpid=93116&utm_source=google&utm_medium=cpc&utm_campaign=(RO:eMAG!)_3P_NO_SALES_%3e_Jucarii_hobby&utm_content=111476631565&gad_source=1&gclid=CjwKCAjw88yxBhBWEiwA7cm6pRCxklxkpkhCiPunMVr4_dPU2-iwMSfpSF-6S8OcYMyXWCNHiOyHPxoCQGIQAvD_BwE) |
```
Comment (alexandruradovici) [inline docs/project/2024/Ioana_maria.cristea/index.md:87]: The link to the DC motor is not present.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/Ioana_maria.cristea/index.md (hunk 6 above)
Comment (Ella23411) [reply docs/project/2024/Ioana_maria.cristea/index.md]: I changed the link.

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**